config/packages/test/translation.yaml

```diff
@@ -0,0 +1,3 @@
+framework:
+    translator:
+        default_path: '%kernel.project_dir%/tests/translations'
```

Comment:
does this folder actually exist?

Reply:
No, `/dev/null` is probably better.

Reply:
`file_exists('/dev/null')` is `true`. 🤔

Reply:
It exists (on Linux), but it's not a regular file:
```
$ test -f /dev/null
$ echo $?
1
$ test -e /dev/null                  
$ echo $?                            
0
$ ls -l /dev/null 
crw-rw-rw- 1 root root 1, 3 Mar 29 08:47 /dev/null
```